lib/src/ui/screens/thankyou/thankyou.dart

```diff
@@ -0,0 +1,91 @@
+import 'package:covidnearme/src/ui/widgets/network_unavailable_banner.dart';
+import 'package:covidnearme/src/ui/widgets/share.dart';
+import 'package:flutter/material.dart';
+import 'package:font_awesome_flutter/font_awesome_flutter.dart';
+
+import 'package:covidnearme/src/l10n/app_localizations.dart';
+import 'package:covidnearme/src/ui/widgets/scrollable_body.dart';
+import 'package:covidnearme/src/ui/widgets/stay_safe.dart';
+
+class ThankYouScreen extends StatefulWidget {
+  static const routeName = '/thankyou';
```

Comment:
Could this be nested under the Symptom Report route? I'm not sure whether snake or kebab case is more appropriate here.
```suggestion
  static const routeName = 'symptom-report/thankyou';
```